connectors/connectors_cli.py

```diff
@@ -45,7 +45,10 @@ def load_config(ctx, config):
 
 
 # Main group
-@click.group(invoke_without_command=True)
+@click.group(
+    invoke_without_command=True,
+    context_settings={"help_option_names": ["-h", "--help"]},
```

Comment:
IIRC, this adds `-h` for the top-level command, but not the subcommands? I couldn't find a clean way to add it all the way through the command tree.